src/ray/gcs/accessor.h

```diff
@@ -232,6 +232,16 @@ class TaskInfoAccessor {
   virtual Status AsyncUnsubscribeTaskLease(const TaskID &task_id,
                                            const StatusCallback &done) = 0;
 
+  /// Updata task reconstruction to GCS asynchronously.
+  ///
+  /// \param data_ptr The task reconstruction that will be added to GCS.
+  /// \param callback Callback that will be called after task reconstruction
+  /// has been added to GCS.
+  /// \return Status
+  virtual Status AsyncUpdateTaskReconstruction(
```

Comment:
This interface is still coupled with the implementation. It'd be better to change it to something like `AttemptTaskReconstruction`. 
But considering we're going to deprecate task reconstruction table soon, this way is also fine.

Reply:
done